### PR TITLE
Bench-173: Aligining Category with API spec

### PR DESCRIPTION
**Pull request by oleksiiRybak**

### CODE REVIEW

File: src/main/java/com/answerdigital/answerking/config/SecurityConfig.java [hunk 1]
```diff
@@ -44,26 +45,16 @@ public SecurityFilterChain securityFilterChain(final HttpSecurity http) throws E
...
-                .authorities(COMMON_ROLE)
-                .build();
-
-        final UserDetails george = User.withUsername("george")
-                .password(COMMON_PASSWORD)
-                .authorities(COMMON_ROLE)
-                .build();
-
-        return new InMemoryUserDetailsManager(paul, john, ringo, george);
+        return new InMemoryUserDetailsManager(
```
Comment (AndyFlintAnswerDigital) [inline src/main/java/com/answerdigital/answerking/config/SecurityConfig.java:48]: Big fan of this.

File: src/main/java/com/answerdigital/answerking/controller/CategoryController.java [hunk 2]
```diff
@@ -67,19 +67,19 @@ public ResponseEntity<Collection<Category>> getAllCategories() {
         return new ResponseEntity<>(categories, categories.isEmpty() ? HttpStatus.NO_CONTENT : HttpStatus.OK);
     }
 
-    @Operation(summary = "Get a single category")
+    @Operation(summary = "Get a single category.")
     @ApiResponses(value = {
         @ApiResponse(responseCode = "200", description = "When the category with the provided id has been found.",
                 content = { @Content(mediaType = "application/json", schema = @Schema(implementation = Category.class)) }),
-        @ApiResponse(responseCode = "404", description = "When the category with the given id does not exist",
+        @ApiResponse(responseCode = "404", description = "When the category with the given id does not exist.",
```
Comment (AndyFlintAnswerDigital) [inline src/main/java/com/answerdigital/answerking/controller/CategoryController.java:74]: This isn't aligned with the spec, but I'd argue this is a fault with the spec.

File: src/test/java/com/answerdigital/answerking/controller/CategoryControllerTest.java [hunk 3]
```diff
@@ -63,6 +68,9 @@ void addCategoryTest() throws Exception {
         final var addCategoryRequest =  new AddCategoryRequest("random name", "random description");
         final var category = new Category(addCategoryRequest);
         final var categoryRequest = "{\"name\": \"random name\",\"description\": \"random description\"}";
+        String testDate = ZonedDateTime.now( ZoneId.of( "Etc/UTC" ) )
```
Comment (AndyFlintAnswerDigital) [inline src/test/java/com/answerdigital/answerking/controller/CategoryControllerTest.java:71]: Should this be final?

**Review by AndyFlintAnswerDigital — CHANGES_REQUESTED** [review]

File: src/main/java/com/answerdigital/answerking/controller/CategoryController.java (hunk 2 above)
Comment (oleksiiRybak) [reply src/main/java/com/answerdigital/answerking/controller/CategoryController.java:74]: for this docs we will have to address it in a different PR.

**Review by AndyFlintAnswerDigital — APPROVED** [review]

**Pull request merged**